contracts/farm/BuyBackAndBurn.sol

```diff
@@ -22,7 +22,7 @@ contract BuyBackAndBurn is Upgradeable_0_8 {
     address public constant USDC = 0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174;
     address public constant P125 = 0x83000597e8420aD7e9EDD410b2883Df1b83823cF;
     IUniswapV3SwapRouter public constant SWAP_ROUTER =
-        IUniswapV3SwapRouter(0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45); //uni v3
+        IUniswapV3SwapRouter(0xE592427A0AEce92De3Edee1F18E0157C05861564); //uni v3
```

Comment:
@Drypto13 why do we have other swap router here? and both labeled v3

Reply:
it is replacing the swap router with the one that uses deadline on code so it compiles properly